### PR TITLE
zest: allow to clear the Zest panel

### PR DESCRIPTION
**Pull request by thc202**
Change ZestResultsPanel to show a button that clears the results of an
executed script.
Bump version and update changes in ZapAddOn.xml file.

### CODE REVIEW

**Comment by thc202** [comment]
There's no section for the results panel, I can add one though.

**Comment by kingthorin** [comment]
Not a blocker...

**Comment by thc202** [comment]
OK.

File: src/org/zaproxy/zap/extension/zest/ZestResultsPanel.java [hunk 1]
```diff
@@ -69,14 +73,29 @@ private void initialize() {
...
 		if (zestPanel == null) {
-
-			zestPanel = new javax.swing.JPanel();
-			zestPanel.setLayout(new java.awt.GridBagLayout());
+			zestPanel = new JPanel(new BorderLayout());
 			zestPanel.setName("ZestResultsPanel");
-			zestPanel.add(getJScrollPane(), 
-					LayoutHelper.getGBC(0, 1, 1, 1.0, 1.0, GridBagConstraints.BOTH, new Insets(2,2,2,2)));
+			zestPanel.add(getToolBar(), BorderLayout.PAGE_START);
+			zestPanel.add(getJScrollPane(), BorderLayout.CENTER);
```
Comment (juhakivekas) [inline src/org/zaproxy/zap/extension/zest/ZestResultsPanel.java]: That looks so clean, will reuse :)

Comment (thc202) [reply src/org/zaproxy/zap/extension/zest/ZestResultsPanel.java]: Yeah, it fits this case :)

**Pull request merged**